client/extensions/woocommerce/app/settings/shipping/shipping-labels.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import ExtendedHeader from '../../../components/extended-header';
+import FormFieldSet from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSelect from 'components/forms/form-select';
+import FormToggle from 'components/forms/form-toggle';
+import ShippingCard from './shipping-card';
+
+class ShippingLabels extends Component {
+	constructor( props ) {
+		super( props );
+
+		//TODO: use redux state with real data
+		this.state = {
+			visible: true,
+			cards: [ {
+				selected: true,
+				type: 'VISA',
+				digits: '1234',
+				name: 'Name Surname',
+				date: '12/19'
+			}, {
+				selected: false,
+				type: 'MasterCard',
+				digits: '5678',
+				name: 'Name Surname',
+				date: '01/20'
+			} ]
+		};
+
+		this.onToggle = this.onToggle.bind( this );
+		this.renderCard = this.renderCard.bind( this );
+	}
+
+	onToggle() {
+		this.setState( { visible: ! this.state.visible } );
+	}
+
+	selectCard( index ) {
+		const cards = this.state.cards.map( ( card ) => {
+			return { ...card, selected: false };
+		} );
+
+		cards[ index ].selected = true;
+
+		this.setState( { cards } );
+	}
+
+	renderCard( card, index ) {
+		const onSelect = () => {
+			this.selectCard( index );
+		};
+
+		return ( <ShippingCard
+			key={ index }
+			onSelect={ onSelect }
+			{ ...card } /> );
+	}
+
+	render() {
+		const { translate } = this.props;
+
+		return (
+			<div>
+				<ExtendedHeader
+					label={ translate( 'Shipping Labels' ) }
+					description={ translate( 'Print shipping labels yourself.' ) }>
+					<FormToggle onChange={ this.onToggle } checked={ this.state.visible } />
+				</ExtendedHeader>
+				<Card className={ classNames( 'shipping__labels-container', { hidden: ! this.state.visible } ) }>
+					<FormFieldSet>
+						<FormLabel
+							className="shipping__labels-paper-size"
+							htmlFor="paper-size">
+							{ translate( 'Paper size' ) }
+						</FormLabel>
+						<FormSelect name="paper-size">
+							<option>{ translate( 'Letter' ) }</option>
+							<option>{ translate( 'Legal' ) }</option>
+							<option>{ translate( 'Label (4"x6")' ) }</option>
+							<option>{ translate( 'A4' ) }</option>
+						</FormSelect>
+					</FormFieldSet>
+					<FormFieldSet>
+						<FormLabel
+							className="shipping__cards-label">
+							{ translate( 'Credit card' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **39** times:
`translate( 'Credit Card' )` *ES Score: 16*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/13559-shipping-ui)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).